maximum-cut/tests/test_maximum_cut.py

```diff
@@ -2,7 +2,7 @@
 import subprocess
 import unittest
 
-# /path/to/demos/pipelines/tests/test_maximum_cut.py
+# /path/to/demos/maximum-cut/tests/test_maximum_cut.py
```

Comment:
probably not part of this PR?

Reply:
I can remove it if you want me to, just seemed like a quick fix to do while I was here :)

Reply:
I was worried it might be a stowaway. If you're aware of it, then that's fine.